### PR TITLE
VMware documentation changes

### PR DESCRIPTION
**Pull request by kalenarndt**
## Labels

- [ x ] inaccuracy
- [ x  ] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md [hunk 1]
```diff
@@ -179,10 +179,10 @@ See [the Upgrades section](../../admin/upgrades.html) of the documentation. For
...
-VMWare hypervisor provides a high level of resilience in various cases
-of failure (at the server hardware layer through vMotion and at the network layer through virtual distributed
-networking.) In addition, having ESXi failover to a DR datacenter
-provides recovery in the case of a total data center outage See the Disaster Recovery section.
+VMware vSphere provides a high level of resilience in various cases
+of failure (at the server hardware layer through vSphere High Availablilty (HA) and at the network layer through virtual distributed
+switching.) In addition, having workloads failover to a DR datacenter with tools like VMware Site Recovery Manager or utilizing stretched clusters
+provides recovery in the case of a total data center outage. See the Disaster Recovery section.
 
 Additional failover can be provide by the Active/Active installation mode.
```
Comment (practicalint) [inline content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:187]:
```suggestion
Additional failover can be provided by the Active/Active installation mode.
```

**Review by practicalint — COMMENTED** [review]
Excellent corrections on product terminology and proper casing! Those of us that have maintained this in the past will not claim ignorance, but probably laziness for having not changed the ones that were done from the beginning.
I and others that have made some updates are experienced in VMware, but from past lives and not as current and knowledgeable as you. So take that into consideration on the comments. I just want to make sure we're giving viable architecture options without being too prescriptive like requiring particular add-on features or not being clear how things can be done.
Along those lines, it would be much appreciated if you could review and give Mark Lewis and I some feedback on the VMware tabs of the TFE backup Recommended Pattern which is published in Learn https://learn.hashicorp.com/tutorials/terraform/pattern-backups?in=terraform/recommended-patterns and the soon to be published Recovery RP - preview link: https://learn-git-enable-sso-tfe-saml-okta-hashicorp.vercel.app/tutorials/terraform/enable-sso-saml-tfe-okta?in=terraform/enterprise

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md [hunk 2]
```diff
@@ -179,10 +179,10 @@ See [the Upgrades section](../../admin/upgrades.html) of the documentation. For
...
 ### Failure Scenarios
 
-VMWare hypervisor provides a high level of resilience in various cases
-of failure (at the server hardware layer through vMotion and at the network layer through virtual distributed
-networking.) In addition, having ESXi failover to a DR datacenter
-provides recovery in the case of a total data center outage See the Disaster Recovery section.
+VMware vSphere provides a high level of resilience in various cases
+of failure (at the server hardware layer through vSphere High Availablilty (HA) and at the network layer through virtual distributed
+switching.) In addition, having workloads failover to a DR datacenter with tools like VMware Site Recovery Manager or utilizing stretched clusters
+provides recovery in the case of a total data center outage. See the Disaster Recovery section.
```
[the remark this answers is not in the document]
Comment (kalenarndt) [reply content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:185]: 1. HA is the backing technology that is set on a vSphere cluster that restarts VMs on a designated physical ESXi host if the one that they are running on fails. vMotion only happens if DRS (Distributed Resources Scheduler) has a recommendation to migrate the workload to a host that has more available resources, a user manually migrates the workload, or the host that the workload was on was put in maintenance mode. 

2. I would say that HA is a requirement and this is enabled in most if not all vSphere clusters. 

3. I always refer to VMs as workloads but we can rephrase this to something else more specific if you would like. 

4. The callout to SRM or other products is really to have the customer be aware that they can use 3rd party tools to replicate the LUN where the VMs reside or the VMs themselves so that way they can have automated failover processes which replicate the running state of the workloads. This handles all of the replication.

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md [hunk 3]
```diff
@@ -192,11 +192,12 @@ Should the *TFE-main* server fail, it can
...
-#### Single ESXi Server Failure
+#### Single ESXi Host Failure
 
-In the event of a single ESXi server failure, ESXi will vMotion the Terraform Enterprise virtual
-machine to a functioning ESXi host. This typically does not result in any
-visible outage to the end-user if VM Fault Tolerance has been configured. As with all applications, you may see a small interruption while the VM boots on the new host if Fault Tolerance is not configured.
+In the event of a single ESXi host failure, vSphere HA will restart the Terraform Enterprise virtual
+machine to a functioning ESXi host in the cluster. This restart can take up to 30 seconds for the failed virtual machine to come back online on a healthy host within the cluster.
+
+If VMware vSphere Fault Tolerance (FT) has been configured for the TFE server, the failover does not result in any visiable outage to the end user.
```
[the remark this answers is not in the document]
Comment (kalenarndt) [reply content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:200]: Yes this would be based on the general recommendation that the customer use vSphere HA at the cluster (This is a VMware base recommendation for any compute cluster) and automated restarts would take place if a host fails. 

We mentioned FT in the original doc (I have rarely seen people use it when I worked at VMware) so I kept it as a 0 downtime type option but this would really be a corner case.

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md [hunk 4]
```diff
@@ -179,24 +179,25 @@ See [the Upgrades section](../../admin/upgrades.html) of the documentation. For
...
-of failure (at the server hardware layer through vMotion and at the network layer through virtual distributed
-networking.) In addition, having ESXi failover to a DR datacenter
-provides recovery in the case of a total data center outage See the Disaster Recovery section.
+VMware vSphere provides a high level of resilience in various cases
+of failure (at the server hardware layer through vSphere High Availablilty (HA) and at the network layer through virtual distributed
+switching.) In addition, having workloads failover to a DR datacenter with tools like VMware Site Recovery Manager or utilizing stretched clusters
+provides recovery in the case of a total data center outage. See the Disaster Recovery section.
 
-Additional failover can be provide by the Active/Active installation mode.
+Additional failover can be provided by the Active/Active installation mode.
```
Comment (laurapacilio) [inline content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:187]:
```suggestion
The Active/Active installation mode can provide additional failover.
```

**Review by laurapacilio — APPROVED** [review]
Thank you so much for doing this! I had one tiny wording suggestion, but these changes look good to me!  However, this is outside my area of expertise, so I defer to @practicalint for the final approval from a technical standpoint :-)

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (practicalint) [reply content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:185]:
```suggestion
VMware vSphere provides a high level of resilience in various cases
of failure, such as at the server hardware layer through vSphere High Availability (HA) and at the network layer through virtual distributed
switching. In addition, employing tools such as VMware Site Recovery Manager or utilizing stretched clusters
can assist in recovery in the case of a total data center to support failover to a DR datacenter. See the Disaster Recovery section.
```

[the remark this answers is not in the document]
Comment (practicalint) [reply content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:185]: Did a little rewording to soften the impression that an automatic DR failover can always be done just through the facilities and to better highlight add-on tools would be needed.

File: content/source/docs/enterprise/before-installing/reference-architecture/vmware.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (practicalint) [reply content/source/docs/enterprise/before-installing/reference-architecture/vmware.md:200]: OK, that answered my concerns.

**Review by practicalint — APPROVED** [review]
lgtm

**Comment by laurapacilio** [comment]
@kalenarndt feel free to merge when ready 🚀
